s3torchbenchmarking/tst/test_compatibility.py

```diff
@@ -0,0 +1,5 @@
+def test_imports():
```

Comment:
Not blocking: This is a good start for tests, but in the future it would be great to include more tests to ensure the functionality is working properly